lib/failure.sh

```diff
@@ -718,6 +718,23 @@ log_other_failures() {
     return 0
   fi
 
+  if grep -q "npm error code EUSAGE" "$log_file"; then
+    if grep -q "Please update your lock file" "$log_file"; then
+      mcount "failures.npm-lockfile-out-of-sync"
```

Comment:
mcount is a no-op post Librato sunset

```suggestion
```

Reply:
Understood. I'm going to remove these as part of a separate work item.